w10h01/test/pgdp/trials/TrialOfTheDreamsTest.java

```diff
@@ -0,0 +1,89 @@
+package pgdp.trials;
+
+import java.util.function.Function;
+import java.security.SecureRandom;
```

Comment:
Changed import respectively. See L67:

https://github.com/MaximilianAnzinger/pgdp2223-tests/blob/df042e1d0e18e3a7dea7e855cf2e51cab0a6dd6d/w10h01/test/pgdp/trials/TrialOfTheDreamsTest.java#L67

```suggestion
import java.util.Random;
```